.github/workflows/continuous-integration-workflow.yml

```diff
@@ -96,11 +96,11 @@ jobs:
           path: ~/.gradle/caches
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
       - name: Build with Gradle
-        run: |
-          set GRADLE_ENTERPRISE_CACHE_USERNAME="$GRADLE_ENTERPRISE_CACHE_USER"
-          set GRADLE_ENTERPRISE_CACHE_PASSWORD="$GRADLE_ENTERPRISE_CACHE_PASSWORD"
-          set GRADLE_ENTERPRISE_ACCESS_KEY="$GRADLE_ENTERPRISE_SECRET_ACCESS_KEY"
-          ./gradlew clean build --continue -PartifactoryUsername="$ARTIFACTORY_USERNAME" -PartifactoryPassword="$ARTIFACTORY_PASSWORD"
+        env:
```

Comment:
Since this is now OS-independent it might be worth it to combine `build_jdk_11` and `build_windows` like so:

```
runs-on: [ubuntu-latest, windows-latest]
```

Will you please create a ticket to take a closer look at that?

Reply:
Sounds good, I create ticket #9790 for that.

Reply:
Hi @jzheaux, would you please review this PR again as I have made some changes since we created #9790 ?